src/ontology/mondo-edit.obo

```diff
@@ -139116,6 +139116,8 @@ synonym: "acoustic schwannomas bilateral" RELATED [GARD:0007193]
 synonym: "acoustic Schwannomas, bilateral" RELATED [OMIM:101000]
 synonym: "bilateral acoustic neurofibromatosis" EXACT [NCIT:C3274, OMIM:101000, Orphanet:637]
 synonym: "central neurofibromatosis" EXACT [NCIT:C3274, Orphanet:637]
+synonym: "full neurofibromatosis type 2" EXACT [Orphanet:637]
```

Comment:
Please add the dbxef that was requested on the ticket: https://www.ncbi.nlm.nih.gov/books/NBK1201/

Reply:
done